### DISCUSSION TITLE
RequestError [HttpError]: Resource not accessible by integration  status: 403 : PSScriptAnalyzer

### DISCUSSION THREAD

**Discussion by unifiedtechnicians** [KelvinTegelaar/CIPP#1560] [Q&A]
I am curretnly facing an issues where the PSScriptAnalyzer fails any ideas on how to fix this or is this an ongoing issue

![image](https://github.com/KelvinTegelaar/CIPP/assets/88386708/70e8a9ea-a22f-4db9-95ef-e4776f84f935)

**Comment by dlpwits** [comment]
We are experiencing the same thing under the 'Upload SARIF results file' section. Initially, with the 403 error: resource not accessible by integration we were getting, we believed this was an API permissions issue. However, even after adjustments were made, we are still showing the PSScriptAnalyzer failing...

**Comment by Priet** [comment]
I have the same error: I can't upgrade to v4 due to this error.

I checked the permissions listed on https://docs.cipp.app/setup/installation/permissions and made sure the CIPP-SAM application has al those permissions.

Someone got an idea how to fix this?

Edit: somehow I fixed this by going to the Function App in Azure, go to Deployment Center and click Sync. Took a few minutes and the webinterface now shows the API version is up-to-date.
